packages/create-sitecore-jss/src/templates/nextjs/src/byoc/index.ts

```diff
@@ -1,17 +1,20 @@
-// This is an out-of-box bundler for External components (BYOC)
-// It enables registering components in client-only or SSR/hybrid contexts
-// It's recommended to not modify this file - please add BYOC imports in corresponding index.*.ts files instead
+import * as FEAAS from '@sitecore-feaas/clientside/react';
+/**
+ * This is an out-of-box bundler for External components (BYOC) (see Sitecore documentation for more details)
+ * It enables registering components in client-only or SSR/hybrid contexts
+ * It's recommended to not modify this file - please add BYOC imports in corresponding index.*.ts files instead
+ */
 
 // Import your client-only components via client-bundle. Nextjs's dynamic() call will ensure they are only rendered client-side
 import dynamic from 'next/dynamic';
 const ClientBundle = dynamic(() => import('./index.client'), {
   ssr: false,
 });
+FEAAS.enableNextClientsideComponents(dynamic, ClientBundle);
 
-// Import your SSR/hybrid components via index.hybrid.ts
+// Import your hybrid (server rendering with client hydration) components via index.hybrid.ts
 import './index.hybrid';
 
 // As long as client bundle is exported and rendered on page (as an empty element), client-only BYOC components are registered and become available
```

Comment:
Does this comment need some updating with the change? There's a bit of magic now between the `FEAAS.enableNextClientsideComponents` above and `FEAAS.ExternalComponentBundle` below.

Reply:
Hybrid import is unaffected - but it makes sense to update the client import comments, or move the enableClientside call